### PR TITLE
Fix CI failure for requiring the long package.

### PR DESCRIPTION
**Pull request by pyu10055**
Fix https://github.com/tensorflow/tfjs/issues/5080

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5085)

### CODE REVIEW

File: tfjs-core/src/hash_util.ts [hunk 1]
```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  * =============================================================================
  */
-// tslint:disable-next-line:no-require-imports variable-name
-const Long = require('long');
+// Workaround for allowing cjs module to be included in bundle created by
+// rollup.
+import * as LongExports from 'long';
+// tslint:disable-next-line
+const Long = (LongExports as any).default || LongExports;
```
Comment (mattsoulanille) [inline tfjs-core/src/hash_util.ts:21]: IIRC, this is what 'esModuleInterop' does, so this approach looks good given the issues we've had with enabling 'esModuleInterop'.

Is the type of `Long` `any` now? We might want to cast it to `typeof Long`.

File: tfjs/rollup.config.js [hunk 2]
```diff
@@ -133,6 +133,7 @@ module.exports = cmdOptions => {
     },
     tsCompilerOptions: {target: 'es5'},
     external: [
+      'long',
```
Comment (mattsoulanille) [inline tfjs/rollup.config.js:136]: Does this mean we're not including `long` in the bundle? Do we need it in the browser?

**Review by mattsoulanille — CHANGES_REQUESTED** [review]

**Review by pyu10055 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5085)* status: 0 of 1 approvals obtained (waiting on @lina128, @mattsoulanille, and @tafsiri)

___
*[tfjs/rollup.config.js, line 136 at r1](https://reviewable.io/reviews/tensorflow/tfjs/5085#-Ma-liEw4FfiqSy8lXp2-r1-136:-Ma09t3mCb2F1dbH6Xf0:b-ugp3zu) ([raw file](https://github.com/tensorflow/tfjs/blob/5ee7874b8e6a46c14b47fc3b0ccef39bf2f2c610/tfjs/rollup.config.js#L136)):*
<details><summary><i>Previously, mattsoulanille (Matthew Soulanille) wrote…</i></summary><blockquote>

Does this mean we're not including `long` in the bundle? Do we need it in the browser?
</blockquote></details>

this is only for node.js not for the browser js files.
___
*[tfjs-core/src/hash_util.ts, line 21 at r1](https://reviewable.io/reviews/tensorflow/tfjs/5085#-Ma-liEw4FfiqSy8lXp1-r1-21:-Ma0IcI8EC9AlYtFrw43:b-896fix) ([raw file](https://github.com/tensorflow/tfjs/blob/5ee7874b8e6a46c14b47fc3b0ccef39bf2f2c610/tfjs-core/src/hash_util.ts#L21)):*
<details><summary><i>Previously, mattsoulanille (Matthew Soulanille) wrote…</i></summary><blockquote>

IIRC, this is what 'esModuleInterop' does, so this approach looks good given the issues we've had with enabling 'esModuleInterop'.

Is the type of `Long` `any` now? We might want to cast it to `typeof Long`.
</blockquote></details>

Done.

**Review by mattsoulanille — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5085)* status: 0 of 1 approvals obtained (waiting on @lina128 and @tafsiri)

___
*[tfjs/rollup.config.js, line 136 at r1](https://reviewable.io/reviews/tensorflow/tfjs/5085#-Ma-liEw4FfiqSy8lXp2-r1-136:-Ma0LeEZ5pJyWx4ZYg1E:b-1xf9ir) ([raw file](https://github.com/tensorflow/tfjs/blob/5ee7874b8e6a46c14b47fc3b0ccef39bf2f2c610/tfjs/rollup.config.js#L136)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

this is only for node.js not for the browser js files.
</blockquote></details>

Ah, okay!
___
*[tfjs-core/src/hash_util.ts, line 21 at r1](https://reviewable.io/reviews/tensorflow/tfjs/5085#-Ma-liEw4FfiqSy8lXp1-r1-21:-Ma0M2kqBGiNmPL0lR2c:b-l7807y) ([raw file](https://github.com/tensorflow/tfjs/blob/5ee7874b8e6a46c14b47fc3b0ccef39bf2f2c610/tfjs-core/src/hash_util.ts#L21)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Thanks! Just wondering why we had to rename `Long` to `LongNS`, but LGTM besides that nit.

**Review by pyu10055 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5085)* status: 0 of 1 approvals obtained (waiting on @lina128 and @tafsiri)

___
*[tfjs-core/src/hash_util.ts, line 21 at r1](https://reviewable.io/reviews/tensorflow/tfjs/5085#-Ma-liEw4FfiqSy8lXp1-r1-21:-Ma0WFO9EkudJFHtnr0r:bktgys5) ([raw file](https://github.com/tensorflow/tfjs/blob/5ee7874b8e6a46c14b47fc3b0ccef39bf2f2c610/tfjs-core/src/hash_util.ts#L21)):*
<details><summary><i>Previously, mattsoulanille (Matthew Soulanille) wrote…</i></summary><blockquote>

Thanks! Just wondering why we had to rename `Long` to `LongNS`, but LGTM besides that nit.
</blockquote></details>

thanks, move back to Long

**Review by lina128 — APPROVED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5085#-:-Ma0YmcE-pqYHD_ia_AB:bb6ywmx)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @tafsiri)

**Review by mattsoulanille — APPROVED** [review]
Sorry, meant to LGTM in my last comments.

> Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5085#-:-Ma0_vLL8J4fFfnNbFJv:bazc0vh)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @tafsiri)

**Pull request merged**